lib/life_game/hoge.rb

```diff
@@ -2,4 +2,39 @@ class Hoge
   def hello
     puts "hello"
   end
+
+  def hello2
+    puts 'hello2'
+  end
+
+  def hello3
+   puts 'hello2'
+  end
+  def hello3
+   puts 'hello2'
+  end
+  def hello3
+   puts 'hello2'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.